modules/govuk/manifests/apps/rummager.pp

```diff
@@ -52,10 +67,22 @@
     ',
   }
 
+  govuk::app::envvar::rabbitmq { 'rummager':
+    hosts    => $rabbitmq_hosts,
+    user     => $rabbitmq_user,
+    password => $rabbitmq_password,
+  }
+
   govuk::procfile::worker { 'rummager':
     enable_service => $enable_procfile_worker,
   }
 
+  govuk::procfile::worker { 'rummager-publishing-api-document-indexer':
+    setenv_as      => 'rummager',
+    enable_service => $enable_procfile_worker,
```

Comment:
I think we want to feature-flag this separately from the existing procfile worker, so we can initially turn it on only in `integration` while we test it out.  So `enable_service => $enable_publishing_api_document_indexer` and around line 38 set a default parameter for that to false.  Then we also need to modify `hieradata/integration.yaml` to set that parameter to true in the `integration` environment (by adding a line something like `govuk::apps::rummager::enable_publishing_api_document_indexer: true`)